README.md

```diff
@@ -273,6 +270,21 @@ In the CORS section of the file you can find the next options:
 - **allow_credentials** (boolean)
   
   Represent the **Access-Control-Allow-Credentials header** you must indicate if true or false
+  
+## Imposter configuration file 
+Imposter file consists of JSON array of imposters. Each imposter can have two sections: `request` that describes request and `response`.
+
+### Request section
+
+### Response section
+Response section describes response that imposter server will send on matching request. Response can have next fields:
+
+#### delay
+Time that server waits before response. This can help simulate network issues, or server high load. You must write delay as string with postfix indicating time unit (see [this](https://golang.org/pkg/time/#ParseDuration) for more info about actual format). Also you can specify minimum and maximum delays using separator ':', server respond delay will be choosen at random between this values.
+```
+"delay": "1s:3m"
+```
+Default value is no delay at all.
```

Comment:
There are an example on the section ##Using Killgrave, with a yaml file, you would need to add in this example your delay and before of the paragraph `The option cors still...` add your text:
<pre>
The `delay` option is a time that server waits before response. This can help simulate network issues, or server high load. You must write delay as string with postfix indicating time unit (see [this](https://golang.org/pkg/time/#ParseDuration) for more info about actual format). Also you can specify minimum and maximum delays using separator ':', server respond delay will be choosen at random between this values.

Default value is no delay at all.
<pre>